ubuntu-provision.sh

```diff
@@ -0,0 +1,277 @@
+#!/bin/bash
```

Comment:
Let's update the Wiki/Readme with links to those scripts since we are simplifying the prerequisites for installation process now.

Reply:
Done